recipes-openxt/xenclient/xenmgr/files/xenstore-init-extra

```diff
@@ -1,5 +1,8 @@
 #!/bin/sh
 
+# xl toolstack domain id
+xenstore-write 'domid' 0
```

Comment:
I wasn't familiar with that tool, but yes, it looks like it is intended to do exactly that.